build/variables.yml

```diff
@@ -69,9 +72,17 @@
   # Flutter version.
   FlutterVersion: '3.22.1'
 
+  # Docker container to run MobSF.
+  # Version.
```

Comment:
I think you can remove the line that just says "Version." which doesn't add much value.